src/libertem/api.py

```diff
@@ -533,16 +543,33 @@ def run_udf(self,
             Restrict the back-end to a subset of the capabilities of the UDF.
             This can be useful for testing hybrid UDFs.
 
+        plots : None or True or List[List[str]] or List[LivePlot]
```

Comment:
The "List[List[str]] or List[LivePlot]" could use a bit more documentation / explanation. Is there other documentation that could be linked? At least the following items:

None: Don't plot anything
True: Plot all UDF buffers, currently only 2D buffers
List[List[str]]: List of UDF buffer names to plot for each of the UDFs passed via the `udf` parameter
List[LivePlot]: LivePlot instance for each UDF, see TODO for more information on live plots.